### PR TITLE
upgrade to Rust 2018 edition

### PR DESCRIPTION
**Pull request by brayniac**
* changes to upgrade to Rust 2018 edition

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -48,7 +37,7 @@ pub mod current {
     #[cfg(target_arch = "x86")]
     pub use bits32::*;
```
Comment (gz) [inline src/lib.rs]:
```suggestion
    pub use crate::bits32::*;
```

Comment (brayniac) [reply src/lib.rs]: good catch! only tested on x86_64 =)

**Comment by brayniac** [comment]
Once this is landed and published to crates.io - will have similar changes to rust-perfcnt which address the currently failing build and upgrades that to 2018 edition as well

**Comment by gz** [comment]
Great the perfcount functionality definitely needs some overhaul. I will publish a new version right away.

**Comment by gz** [comment]
0.15 has been published on crates.io

**Pull request merged**